subcommittee/coding-guidelines/meetings/19-November-2024/agenda.md

```diff
@@ -0,0 +1,40 @@
+# Coding Guidelines Subcommittee Meeting on 2024-11-19
+
```

Comment:
Might be good to put the UTC time of the meeting in the agenda too.

Reply:
Good idea. Done.